lib/chipmunk_bagger.rb

```diff
@@ -40,51 +48,20 @@ def common_tags
     }
   end
 
-  def content_specific_tags
-  
-  end
-
   def move_files_to_data
-    bag_data_dir = File.join(bag_path,"data")
-    return if File.exists?(bag_data_dir)
-
-    files_to_move = Dir.glob(File.join(bag_path,"*"))
-    FileUtils.mkdir(bag_data_dir)
-    FileUtils.mv(files_to_move,File.join(bag_path,"data"))
-  end
-
-
-  def audio_tags
-    # extract metadata path from mets
-    doc = Nokogiri::XML(File.open(File.join(bag_path,"data","mets.xml")))
-
-    # try MARC
-    marc_href_attr = doc.xpath("//mets:mdRef[@MDTYPE='MARC']/@xlink:href").first
-    
-    if marc_href_attr
-      {'Metadata-URL': download_marc(marc_href_attr.value),
-       'Metadata-Type': 'MARC'}
-    else
-      raise ArgumentError,"No linked MARC metadata found in mets.xml"
+    Find.find(src_path) do |file|
+      bag.add_file_by_moving(file.sub(src_path,''),file) if File.file?(file)
```

Comment:
This line took a minute to figure out. Would it be better to just pass the dir (src_path) and the filename?

Thinking this through with @daaang, we are wondering if the relative paths are mirrored by design/requirement. 

Do we ever envision calling things like `bag.add_file_by_moving("new/fun/path.xml", "/base/src_dir/old/path.xml")`? Because if not, we could use the [base|rel] construction: `bag.add_file_by_moving("/base/src_dir", "old/path.xml")`.